server/openai/generateItenerary.js

```diff
@@ -0,0 +1,56 @@
+const openaiClient = require('./openaiClient')
+
+async function generateItinerary(constraints) {
+  try {
+    const conversation = [
+      {
+        role: 'system',
+        content: `You are an AI that generates travel itineraries. Respond with ONLY JSON that contains the plan for each stage. 
+        ONLY respond with the following format, do not include any descriptions or codeblocks, I should be able to parse the output to a JavaScript Object
+        The response needs to be formatted exactly like the following structure. Add some Food and snack stages as well with an optional: true marker.:
+        '''
+        {
+          plan:[
+            {
+              day: Number
+              bullet_point_summary: String (comma seperated. Can be the names of each place)
+              stages:[
+                {
+                  stage: Number,
+                  place: String,
+                  description: String,
```

Comment:
We don't need address from openapi here, just place name is sufficient. Places API is able to fetch based off of business/attraction name.

Reply:
removed